### PR TITLE
fix: Add partial index selectors to sift

### PR DESCRIPTION
**Pull request by JF-Cozy**
fix https://github.com/cozy/cozy-client/issues/845

### CODE REVIEW

**Comment by ptbrowne** [comment]
Thanks :+1: Can you add a test ?

**Comment by JF-Cozy** [comment]
> Thanks 👍 Can you add a test ?

Oops oui bien sûr

**Review by ptbrowne — APPROVED** [review]

File: packages/cozy-client/src/store/queries.js [hunk 1]
```diff
@@ -139,15 +139,28 @@ export const convert$gtNullSelectors = selector => {
...
+ * @param {object} queryDefinition - A query definition
+ * @returns {object} A query definition selector
+ */
+export const mergeSelectorAndPartialIndex = queryDefinition => ({
+  ...queryDefinition.selector,
+  ...get(queryDefinition, 'partialFilter')
+})
+
 const getSelectorFilterFn = queryDefinition => {
+  console.info('qdef', queryDefinition)
```
Comment (paultranvan) [inline packages/cozy-client/src/store/queries.js:154]: I guess this should removed ;)

Comment (JF-Cozy) [reply packages/cozy-client/src/store/queries.js:154]: https://github.com/cozy/cozy-client/pull/849

**Pull request merged**